applications/mne_scan/libs/scShared/Management/pluginmanager.cpp

```diff
@@ -49,6 +49,7 @@
 #include <../plugins/neuromag/neuromag.h>
 #include <../plugins/babymeg/babymeg.h>
 #include <../plugins/natus/natus.h>
+//#include <../plugins/ftbuffer/ftbuffer.h>
```

Comment:
the ft plugin can be commented in. only plugins which need closed source SDKs are commented out here.